tests/adapter/mongo/test_query.py

```diff
@@ -539,146 +578,151 @@ def test_build_range(adapter):
     assert mongo_query['position'] == {'$lte': end}
     assert mongo_query['end'] == {'$gte': start}
 
-def test_get_overlapping_variant(populated_database, parsed_case):
```

Comment:
Eh, sorry, where did the original overlapping variant test go?

Reply:
I replaced the code of it with new code, because the test was very slow